WordPress/src/main/java/org/wordpress/android/ui/utils/InAppReviewsUtils.kt

```diff
@@ -0,0 +1,15 @@
+package org.wordpress.android.ui.utils
+
+import com.google.android.gms.tasks.Task
+import com.google.android.play.core.review.ReviewException
+import com.google.android.play.core.review.ReviewInfo
+import com.google.android.play.core.review.model.ReviewErrorCode
+import org.wordpress.android.util.AppLog
+
+fun Task<ReviewInfo>.logException() {
```

Comment:
Could you actually move `InAppReviewsUtils` to:

    WordPress/src/main/java/org/wordpress/android/util/extensions/InAppReviewsUtils.kt

Would keep all our extension functions nice and tidy.

Reply:
Good point! Done: 7cbbb26d3a833f40c2db6ef8d8d5b20ac4122134